core/src/main/java/org/owasp/dependencycheck/data/update/NvdApiDataSource.java

```diff
@@ -367,7 +367,9 @@ private boolean processApi() throws UpdateException {
                         if ((ctr % 5) == 0) {
                             //TODO get results per page from the API as it could adjust automatically
                             final double percent = (double) (ctr * resultsPerPage) / max * 100;
-                            LOGGER.info(String.format("Downloaded %,d/%,d (%.0f%%)", ctr * resultsPerPage, max, percent));
+                            if (percent < 100) {
+                                LOGGER.info(String.format("Downloaded %,d/%,d (%.0f%%)", ctr * resultsPerPage, max, percent));
+                            }
```

Comment:
Maybe still write the completion log entry as well, but use just the actual counts and a hardcoded 100% ?

```suggestion
                            if (percent < 100) {
                                LOGGER.info(String.format("Downloaded %,d/%,d (%.0f%%)", ctr * resultsPerPage, max, percent));
                            } else {
                                LOGGER.info(String.format("Downloaded %,d/%,d (%.0f%%)", max, max, 100));
                            }
```

Reply:
The 100% complete is already written out here: https://github.com/jeremylong/DependencyCheck/pull/6899/files#diff-871e872a0687fe3f7ff7af515affbfc9c90ef55c0322f2e9c3a83602065e1715R398

Reply:
ah right...